frontend/packages/dev-console/integration-tests/features/addFlow/create-from-git.feature

```diff
@@ -172,3 +172,33 @@ Feature: Create Application from git form
                   | https://github.com/sclorg/django-ex.git        | django-ex-git-app  | django-ex-git         |
                   | https://github.com/spring-projects/spring-boot | spring-bottom-app  | openshift-quickstarts |
                   | https://github.com/sclorg/nodejs-ex.git        | nodejs-ex-git-app  | nodejs-ex-git         |
+
+
+        @regression @manual
+        Scenario Outline: Dotnet Builder image detection for git url "<git_url>": A-06-TC13
```

Comment:
Shouldn't the scenario outline title be generic to test the context dir flow instead of this scenario bound to Dotnet?

Reply:
We have only one builder image right related to context directory, so i don't think it is to be Generic

Reply:
No,  Context directory is field is where you can provide subfolder name, 
for example: 
In this reposistory https://github.com/sclorg/s2i-nodejs-container, there are many subfolders for each versions, so if you want to specifically use nodejs 14 then in the context dir you need to provide `/14`. It is a generic field and not only dotnet builder image. IMO, this scenario could be a generic but I am fine if you are okay with having dotnet specific test case.

Reply:
yeah I agree context dir field is generic, but I want to make sure this dotnet scenario is covered